fixieai/cli/agent/commands.py

```diff
@@ -107,17 +107,28 @@ def _update_agent_requirements(
     return resolved_requirements
 
 
+def config_already_exists():
+    try:
+        agent_config.load_config()
+        return True
+    except FileNotFoundError:
+        return False
+
+
+default_config = agent_config.AgentConfig()
```

Comment:
nit: consider either wrapping this in a function (and doing `default_config().handle` etc.) or just inlining `agent_config.AgentConfig().handle` to avoid a slightly confusing (mutable) variable in the global scope